equinix/resource_network_device_test.go

```diff
@@ -136,7 +139,7 @@ func TestNetworkDevice_updateResourceData(t *testing.T) {
 	assert.Equal(t, ne.StringValue(inputPrimary.PackageCode), d.Get(networkDeviceSchemaNames["PackageCode"]), "PackageCode matches")
 	assert.Equal(t, ne.StringValue(inputPrimary.Version), d.Get(networkDeviceSchemaNames["Version"]), "Version matches")
 	assert.Equal(t, ne.BoolValue(inputPrimary.IsBYOL), d.Get(networkDeviceSchemaNames["IsBYOL"]), "IsBYOL matches")
-	assert.Equal(t, ne.StringValue(inputPrimary.LicenseToken), d.Get(networkDeviceSchemaNames["LicenseToken"]), "LicenseToken matches")
+	assert.Empty(t, d.Get(networkDeviceSchemaNames["LicenseToken"]), "LicenseToken is empty")
```

Comment:
I'm puzzled by this test - should we expect the LicenseToken to be unchanged from the creation state?

Reply:
These tests are unusual Terraform provider tests.  Usually Terraform tests are constructed with HCL stanzas being applied on top of each and tested after they are applied.

 I think I see why this is, we are not starting with the state used in TestNetworkDevice_createFromResourceData, we are starting from an empty (possibly invalid) state - https://github.com/equinix/terraform-provider-equinix/pull/59/files#diff-b562c6e4659cba3932687df05efa8fc8149a3412bfe2bb8a506bcb787250dc7dR124

Reply:
Yes, License Token is not returned by the GET API as we don't store that information. The token value should remain the same as that value provided during the creation process.

Reply:
> The token value should remain the same as that value provided during the creation process.

And in this test, the initial value is `""`, right?